### ISSUE TITLE
Flashing the FPGA

### ISSUE THREAD

**Issue by Syntec1** [jbilander/ReAgnus#5]
Hi,

I understand this is a work in progress project, I would like to build 2 to test on a A2000 and A500.
unfortunately I had a denied response for the other project.
I am willing to share the info on the build, flash and testing from my end, and yes I'm experienced in building such boards.
Maybe I missed it but I don't not see the code for flashing the FPGA, or is it yet to be created or still in development phase?
If it is not possible to participate then at least I tried :)

**Comment by jbilander** [comment]
Hi, this repository is only the hardware design, the PCB.
Niklas Ekström has been successful at writing HDL for it and making it work in a 10M16SCU169C8G FPGA, you will have to ask him about firmware.
The problem is the Intel Max 10 FPGAs have not been available and are still not available to source from anywhere since over two years now.

**Comment by niklasekstrom** [comment]
Hi guys,

Yeah over a year ago I was hacking on the HDL for ReAgnus and got a modified version of [Minimig's Agnus implementation ](https://github.com/MiSTer-devel/Minimig-AGA_MiSTer/tree/MiSTer/rtl) to run. Here is a picture of it running Silkwork: http://eab.abime.net/showpost.php?p=1544845&postcount=17

The problem I experienced with the ReAgnus design was that I felt it was mechanically unstable. And then as Jörgen mentioned, the Intel Max 10 has been virtually impossible to buy the last year. Both of those issues are addressed by a new design by Jörgen, ReAgnus-MegAChip, which is currently in a private repository since it hasn't been tested yet, but will hopefully be tested "soon".

I am not inclined to release the HDL I have written until we have a version of both hardware and HDL that I believe people can build and be happy with.

If you like then perhaps we can setup a Discord server where we can discuss ReAgnus development, and then maybe that will help move the development along at a faster pace?

**Comment by nonarkitten** [comment]
You weren't "denied" anything @Syntec1

**Comment by Syntec1** [comment]
English is not my native language, Excuse me for expressing it incorrectly.

**Comment by thbreidenbach** [comment]
Hey there, are there any news on the firmware side or in establishing a Discord for Disussion? On Hardware side there could be several options including a removable low profile version without a socket (Using https://www.mouser.de/ProductDetail/575-3016015152127 after removal of the Agnus Socket) This way even a common PLCC Socket can be placed when needed.

**Comment by jbilander** [comment]
@thbreidenbach 

Hi, I think frenchshark is looking into developing a firmware. I have no idea if it will be open source or not and how far he has come. Discussion in Retro Tinkering discord under Technical stuff -> modern-enhanced-or-replacement-parts.

**Comment by nonarkitten** [comment]
He has a github page for it.
https://github.com/fredrequin/AliceRE

**Comment by thbreidenbach** [comment]
Yeah, found that but unfortunately there's no firmware release yet.
